### ISSUE TITLE
Permission denied in set and get methods in IE and Microsoft Edge

### ISSUE THREAD

**Issue by kfuzaylov** [emberjs/ember.js#15589]
I have DOM element and set it to property
```javascript
let link = document.createElement('link');
// insert link into dom
this.set('domElement', link);
```
then remove link from DOM structure and try to get or set this property
```javascript
if (this.get('domElement')) // Permission denied
// Error happens in get method here
// var desc = value !== null && typeof value === 'object' && value.isDescriptor ? value : undefined; 

this.set('domElement', 333); // Permission denied
// Error happens in set method here
// var possibleDesc = obj[keyName];
// if (possibleDesc !== null && typeof possibleDesc === 'object' && possibleDesc.isDescriptor) {
```

**Comment by krisselden** [comment]
@stefanpenner should we use instanceof Descriptor?  We should resurrect the move back to meta

**Comment by bekzod** [comment]
I can move everything to `instanceof Descriptor` @krisselden if you want :)

**Comment by rwjblue** [comment]
Instanceof has issues across realms, not sure if that is a factor for us (fastboot uses a different realm to run app code vs evaluate vendor assets)....

**Comment by pixelhandler** [comment]
@bekzod @kfuzaylov @krisselden @rwjblue is this still an issue, perhaps we should close, what do you think?

**Comment by rwjblue** [comment]
Just double checked, and we still do the check in question due to deprecated “descriptor as a property” fallback.

https://github.com/emberjs/ember.js/blob/10f5625eafa749329c9aa7e1fe57d80601beab71/packages/%40ember/-internals/metal/lib/property_get.ts#L120-L139

https://github.com/emberjs/ember.js/blob/10f5625eafa749329c9aa7e1fe57d80601beab71/packages/%40ember/-internals/meta/lib/meta.ts#L738-L744

**Comment by rwjblue** [comment]
The good news here is that since master is tracking 3.6 and those depreciations are “until 3.5”, we can actually remove the fallback now.

**Comment by rwjblue** [comment]
Closed by #17020
